wagtailvideos/views/chooser.py

```diff
@@ -114,7 +115,12 @@ def chooser_upload(request):
             video.save()
 
             # Reindex the video to make sure all tags are indexed
-            search_index.insert_or_update_object(video)
+            from wagtail.wagtailcore import __version__
+            if __version__.startswith("1.4"):
```

Comment:
I'll make a PR against Wagtail soon to add a `version_info` tuple, so you can do nicer things like `if version_info <= (1, 4)`. This is the only good solution for now though :+1: